src/workshops/applicant-git-1/index.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Git introduction
+description: This workshop is an introduction to using Git for version control; GitHub for hosting a codebase and deploying a website; and VS Code for writing and editing code, as well as version control.
+tags:
+  - workshop
+  - git
+keywords:
+  - git
+  - introduction
+starter: false
+---
+
+## Git
+
+Git is a a language for controlling versions of a project. Git allows you to track changes you make on files across time.
+
+To make an analogy, it's like having a save button on your project. At any time, you can save what you've written and mark that point in time. This means you can review versions of the code or go back to a previous version.
+
+## GitHub
+
+GitHub, although similar in name, is not directly related to Git. GitHub is a platform for storing code online.
+
+Each project is stored in a _repository_ - this is a folder where all the files are contained on GitHub.
+
+GitHub also allows you to host a live version of your website via GitHub Pages. You can share the link with anyone and they can visit websites that you've created!
+
+## Local vs. Remote
+
+We'll describe repositories in terms of local and remote versions.
+
+A _local_ repository is a folder on your computer which you save files to. A _remote_ repository is a folder online which your files can be duplicated to.
+
+## Git Flow
+
+_Git flow_ refers to the process of synchronising code between a local and remote repository.
```

Comment:
I think we should use a different term to describe this to be precise. Git flow/gitflow refers to quite a [specific methodology](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow) that we definitely don't teach/follow/encourage.

We roughly teach Git _Hub_ Flow (https://docs.github.com/en/get-started/quickstart/github-flow), but it seems like this section is more generally referring to just a general "workflow" for working with Git/GitHub.